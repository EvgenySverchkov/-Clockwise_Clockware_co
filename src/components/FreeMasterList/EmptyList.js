import React from "react";
import PropTypes from "prop-types";

const EmptyList = ({children}) => (
    <div className="text-left display-4">
        {children}
    </div>
);

EmptyList.propTypes = {
    children: PropTypes.node.isRequired
}

export default EmptyList;