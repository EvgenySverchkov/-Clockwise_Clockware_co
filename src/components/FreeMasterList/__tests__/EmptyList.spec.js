import React from "react";
import {create} from "react-test-renderer";

import EmptyList from "../EmptyList";

it("Test of <EmptyList/> for <FreeMasterList/>", ()=>{
    let component = create(
        <EmptyList>
            <div>Mock element</div>
        </EmptyList>
    );
    expect(component.toJSON()).toMatchSnapshot();
});