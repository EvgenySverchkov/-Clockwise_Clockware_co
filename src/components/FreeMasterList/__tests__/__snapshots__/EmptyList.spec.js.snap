// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test of <EmptyList/> for <FreeMasterList/> 1`] = `
<div
  className="text-left display-4"
>
  <div>
    Mock element
  </div>
</div>
`;
